spec/features/display_dashboard_spec.rb

```diff
@@ -13,11 +13,6 @@
       expect(page).to have_content "Your Statistics"
     end
 
-    it "lets the user upload files" do
```

Comment:
"Upload" was going to the old way of adding files, now there's only "create work," for which a test already existed.